<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>621</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu Mono</family>
    <pointsize>14</pointsize>
    <italic>true</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="font">
       <font>
        <family>Ubuntu Mono</family>
        <pointsize>14</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="2">
         <widget class="QFrame" name="frame">
          <property name="font">
           <font>
            <family>Ubuntu Mono</family>
            <pointsize>14</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="minimumSize">
                <size>
                 <width>170</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Ubuntu Mono</family>
                 <pointsize>14</pointsize>
                 <italic>true</italic>
                </font>
               </property>
               <property name="text">
                <string>Laser Point Cloud</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_laser">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Ubuntu Mono</family>
                 <pointsize>14</pointsize>
                 <italic>true</italic>
                </font>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_load_laser">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Ubuntu Mono</family>
                 <pointsize>14</pointsize>
                 <italic>true</italic>
                </font>
               </property>
               <property name="text">
                <string>load</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="minimumSize">
                <size>
                 <width>170</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Ubuntu Mono</family>
                 <pointsize>14</pointsize>
                 <italic>true</italic>
                </font>
               </property>
               <property name="text">
                <string>Station Points</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_station">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Ubuntu Mono</family>
                 <pointsize>14</pointsize>
                 <italic>true</italic>
                </font>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_load_station">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Ubuntu Mono</family>
                 <pointsize>14</pointsize>
                 <italic>true</italic>
                </font>
               </property>
               <property name="text">
                <string>load</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QTabWidget" name="tabWidget">
          <property name="font">
           <font>
            <family>Ubuntu Mono</family>
            <pointsize>14</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>3D Visual</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="text">
                 <string>Laser</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QGridLayout" name="layout_3dgraph_laser"/>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_icp">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="text">
                 <string>Before ICP</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QGridLayout" name="layout_3dgraph_same_coord"/>
              </item>
             </layout>
            </item>
            <item row="0" column="2">
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="text">
                 <string>Station</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QGridLayout" name="layout_3dgraph_station"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Points Info</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="text">
                 <string>Laser</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="tab_laser"/>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="text">
                 <string>Initial Station</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="tab_station_init"/>
              </item>
             </layout>
            </item>
            <item row="0" column="2">
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="text">
                 <string>Registered Station</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="tab_station_reg"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Result</string>
           </attribute>
           <widget class="QWidget" name="">
            <property name="geometry">
             <rect>
              <x>9</x>
              <y>9</y>
              <width>311</width>
              <height>381</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>35</height>
                </size>
               </property>
               <property name="text">
                <string>Transform</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableWidget" name="tab_trans"/>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QFrame" name="frame_2">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="1">
            <widget class="QPushButton" name="btn_icp">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>Run ICP</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>907</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
